src/ims-jwt.js

```diff
@@ -25,7 +25,7 @@ function configMissingKeys (configData) {
   }
 
   const missingKeys = []
-  const requiredKeys = ['client_id', 'client_secret', 'technical_account_email', 'technical_account_id', 'meta_scopes', 'ims_org_id', 'private_key']
+  const requiredKeys = ['client_id', 'client_secret', 'technical_account_id', 'meta_scopes', 'ims_org_id', 'private_key']
```

Comment:
will the plugin support a configuration that still has `technical_account_email` in the context? 
Thinking about backward compatibility + being able to generate a token from the .aio jwt config loaded by the app plugin (which still includes the email)

Reply:
it should, since `technical_account_email` being in the context will just be something extra and not used.

Reply:
ok perfect 👍 thx